src/amlaidatatests/tests/test_risk_case_event_table.py

```diff
@@ -240,27 +240,21 @@ def test_DT014_event_order(connection, request):
             test_id="P044",
             value="AML_EXIT",
         ),
-        # TODO: This is not a precise test. We are comparing the number of
-        # values of party_id, risk_case_id which have an AML_PROCESS_START over
-        # the number which have an AML_EXIT. This could be a co-incidence, so we
-        # test this in a few different ways
-        common.VerifyTypedValuePresence(
+        common.VerifyEntitySubset(
             column="type",
             table_config=TABLE_CONFIG,
-            min_proportion=1,
-            group_by=["party_id", "risk_case_id"],
-            compare_group_by_where=lambda t: t.type == "AML_EXIT",
             test_id="P048",
-            value="AML_PROCESS_START",
+            concat=["party_id", "risk_case_id"],
```

Comment:
I chose to change this to `concat` in this test, because it seemed that there were no group by aggregations in the test now that the `concat`/`group_by` columns are just used for joining. Perhaps something like `entity_id_columns` would be clearer?

Reply:
I think we do need a group by expression - it's per risk_case_id and party_id, added a commend below as well.